### PR TITLE
Move CLI form lindera repository

### LINKED ISSUE #1: Move from lindera repository
Move from lindera repository.

### PR DESCRIPTION
**Pull request by mosuka**
Move CLI form lindera repository.

Closes #1

### CODE REVIEW

File: examples/input.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+形態素解析（けいたいそかいせき、Morphological Analysis）とは、文法的な情報の注記の無い自然言語のテキストデータ（文）から、対象言語の文法や、辞書と呼ばれる単語の品詞等の情報にもとづき、形態素（Morpheme, おおまかにいえば、言語で意味を持つ最小単位）の列に分割し、それぞれの形態素の品詞等を判別する作業である。
```
Comment (johtani) [inline examples/input.txt:1]: Why do we have this example file?

**Review by johtani — COMMENTED** [review]
Should we remove the file or use it in README.md ?

File: examples/input.txt (hunk 1 above)
Comment (mosuka) [reply examples/input.txt:1]: That's a quote from Wikipedia.

Comment (mosuka) [reply examples/input.txt:1]: This file is not in use.
I'll delete it.

Comment (mosuka) [reply examples/input.txt]: Deleted.

**Review by johtani — APPROVED** [review]
LGTM!

**Pull request merged**